profiles/common/modules/features/sitemap/sitemap.install

```diff
@@ -85,3 +78,11 @@ function sitemap_update_7120() {
 
   cache_clear_all('variables', 'cache_bootstrap');
 }
+
+/**
+ * NEXTEUROPA-12244: Disable and uninstall xmlsitemap_i18n module.
+ */
+function sitemap_update_7121() {
+  module_disable(array('xmlsitemap_i18n'), FALSE);
```

Comment:
This does not offer any guarantuees that the xmlsitemap_i18n module won't be enabled again by a subsite builder on a new website.